configs/projects/pl-build-tools.rb

```diff
@@ -16,8 +16,12 @@
   # Note that the platform.architecture is the Debian packaging arch,
   # which differs from the platform triple used when compiling:
   platform_triple = "powerpc64le-linux-gnu"
-elsif platform.architecture == "ppc64le" && platform.is_rpm?
```

Comment:
Won't removing the is_rpm? here effect the ubuntu power8 builds?

Reply:
i.e. during the ubuntu build vanagon will match the arch during this if statement and set the triple to redhat.

Reply:
We check "is_deb" above, so the check for RPM was redundant (if we're not deb, we're going to be RPM)

Reply:
Deb-based platforms have a platform.architecture of "ppc64el" instead of "ppc64le", so this shouldn't case problems.

Reply:
Although folding all those cases together might be worthwhile?

Reply:
Oh, I see. looks good then